application/src/main/webapp/vue-app/components/ExoChatMessageList.vue

```diff
@@ -35,6 +35,7 @@
       <span v-show="!newMessagesLoading && (!messages || !messages.length)" class="text">{{ $t('exoplatform.chat.no.messages') }}</span>
     </div>
     <exo-chat-message-composer
+      :composersApplications="composerApplications"
```

Comment:
I think we can avoid changing in this component and access directly to the $root component . could you check that ?